pkg/reconciler/pullsubscription/pullsubscription.go

```diff
@@ -56,7 +56,8 @@ const (
 	// ReconcilerName is the name of the reconciler
 	ReconcilerName = "PullSubscriptions"
 
-	component = "pullsubscriptions"
+	// TODO change to importer once StackDriver MonitorResources are approved.
+	component = "gcpimporter"
```

Comment:
We woould like to avoid using `importer` in the code anywhere.

Reply:
let's sync on this offline... We are proposing a SD monitored resource called importer, so we may need to revisit that as well.